### PR TITLE
pyGeo wrapper fix

### PR DESCRIPTION
**Pull request by ArshSaja**
This PR is to address the failing tests and examples due to the recent changes on the pyGeo mPhys wrapper. The tests and the examples were fixed to work with the current version of pyGeo.

### CODE REVIEW

**Comment by ArshSaja** [comment]
@friedenhe I think 2 files of DAfoam tests were updated in this PR. If it worked for the others, it should be good I suppose.

**Review by bernardopacini — APPROVED** [review]
I just pushed updates to the DAFoam tests and they work locally on my machine. Overall this looks good to me.

@ArshSaja, did you test this all with the OpenMDAO 3.25? If not, I suggest a quick rerun of the tests just to be sure they work (I checked DAFoam with 3.25 already).

**Comment by ArshSaja** [comment]
@bernardopacini Yes the tests were tested with the latest version of OpenMDAO. Thanks @anilyil  and @bernardopacini !

File: tests/regression_tests/test_dafoam_aerostruct.py [hunk 1]
```diff
@@ -187,7 +187,7 @@ def problem_setup(scenario_name, fea_assembler, problem):
         tacs_options = {
             "element_callback": element_callback,
             "problem_setup": problem_setup,
-            "mesh_file": "../input_files/wingbox.bdf",
```
Comment (ArshSaja) [inline tests/regression_tests/test_dafoam_aerostruct.py]: The `wingbox.bdf` should be inside the `input_files`. Isn't it?

**Pull request merged**